WordPress/Classes/ViewRelated/System/FancyAlerts+VerificationPrompt.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  FancyAlerts+VerificationPrompt.swift
+//  WordPress
+//
+//  Created by Jan Klausa on 03.10.17.
+//  Copyright © 2017 WordPress. All rights reserved.
+//
+
+import UIKit
+
+extension FancyAlertViewController {
+
+    private enum VerificationFailureError: Error {
+        case alreadyVerified
+        case unknown
+    }
+
+    public static func verificationPromptController(completion: (() -> Void)?) -> FancyAlertViewController {
+        let resendEmailButton = FancyAlertViewController.Config.ButtonConfig(Strings.resendEmail) { controller, button in
+
+            let managedObjectContext = ContextManager.sharedInstance().mainContext
+            let accountService = AccountService(managedObjectContext: managedObjectContext)
+
+            let submitButton = button as? NUXSubmitButton
+
+            submitButton?.showActivityIndicator(true)
+
+            accountService.requestVerificationEmail({ _ in
+                submitButton?.showActivityIndicator(false)
+                controller.setViewConfiguration(successfullySentVerificationEmailConfig(), animated: true)
+            }, failure: { error in
+                submitButton?.showActivityIndicator(false)
+
+                let localError: VerificationFailureError
```

Comment:
A couple of thoughts.  As a best practice we try to avoid late initialization of constants.  We also try to put a case statement's block on a new line after the case statement for readability. 

In this case.. since there is a single case to evaluate and everything else is handled by the default case... would it simplify things to declare `localError` with `var` and assign `.unknown` (being the default)  then check for `alreadyVerified` via an `if` statement? 

(Did that make sense?)

Reply:
yup! I tend to default to `switch`es, because I like the compiler yelling at me if I forgot to handle a case, but that wouldn't happen here anyway because of the `default`. changed ;)